configs/ruby/GPU_VIPER.py

```diff
@@ -298,7 +300,10 @@ def create(self, options):
 class TCCCntrl(TCC_Controller, CntrlBase):
     def create(self, options, ruby_system, system):
         self.version = self.versionCount()
-        self.L2cache = TCC()
+        self.L2cache = TCC(
+            tagAccessLatency=options.tcc_tag_access_latency,
+            dataAccessLatency=options.tcc_data_access_latency,
+        )
         self.L2cache.create(options)
         self.L2cache.resourceStalls = options.no_tcc_resource_stalls
```

Comment:
Isn't this logic backwards?

Reply:
Sorry, I missed that there was an important, pending question here.  I'm not sure what the issue is though, both options are assigned to the corresponding one?